src/desktop/apps/authentication/helpers.ts

```diff
@@ -185,7 +187,7 @@ const loginUser = async (
     error: (err: any) => void
   }
 ) => {
-  const url = `${sd.APP_URL}${sd.AP.loginPagePath}`
+  const url = `${sd.APP_URL}${sd.AP.loginPagePath}/omg`
```

Comment:
this is likely a test url?

Reply:
I definitely did not add this! And it explains why I was seeing a random url test fail. TY.